openreview/openreview.py

```diff
@@ -67,8 +67,7 @@ def __handle_response(self,response):
 
             return response
         except requests.exceptions.HTTPError as e:
-            for k,v in response.json().iteritems():
-                raise OpenReviewException(str(v))
+            raise OpenReviewException(response.json()['errors'])
```

Comment:
I'm not sure if always you will get an errors property, could you check if exists first? otherwise return the whole json.